.github/CODEOWNERS

```
@@ -1 +1 @@
-* @iDmple @greut @floriansipp
+* @iDmple @floriansipp @jdaln 
```

then, let's keep it simple for now.

```suggestion
* @iDmple @floriansipp
```